packages/fast-markdown-msft-react/src/index.ts

```diff
@@ -0,0 +1,56 @@
+import CreateRule from "./rule";
+import { MarkdownIt } from "markdown-it";
+
+enum ComponentType {
+    heading = "heading",
+    hyperlink = "hyperlink",
+    list = "list",
+    paragraph = "paragraph"
+}
+
+export interface ITokens<T> extends Array<T> {
+    [idx: number]: T;
+}
+
+export interface IToken {
+    children: ITokens<IToken>;
+    tag: string;
+    content: string;
+    info: string;
+    type: string;
+    attrs: string[][];
+    attrPush: (attributes: string[]) => void;
+}
+
+class FastMarkdownIt {
+    constructor(md: MarkdownIt) {
+        md.core.ruler.push("fast", (new CreateRule(md) as any));
+        md.renderer.rules.paragraph_open = function(): string {
+            return `<Typography tag="p" typeLevel={2}>`;
+        };
+        md.renderer.rules.heading_open = function(tokens: ITokens<IToken>, idx: number): string {
```

Comment:
If this is specific to FAST MSFT, should we not use the Heading component? If this isn't specific to MSFT, can you update the ReadMe language?

Reply:
Highlighted, it put it where it wanted. I'll keep an eye out for it.